src/mainboard/pcengines/apu2/board.fmd

```diff
@@ -0,0 +1,18 @@
+FLASH 8M {
+	SI_BIOS@0x0 0x800000 {
+		BOOTORDER(PRESERVE)@0x0 0x1000
+		RW_VPD(PRESERVE)@0x1000 0x4000
```

Comment:
Done in https://github.com/Dasharo/coreboot/compare/adb3266b83b989afa7cf8666263f21b770bccffb..6b4a0b81635b919c6fa21814b3b95c7612fe5a6f

Reply:
@krystian-hebel we will not provide anything else than UEFI in Dasharo. If someone wants to switch to something else, flashing whole image will be required...

Reply:
No idea. It is currently out of scope of the projects. But board.fmd is not used at all right now in the project, so it may have whatever regions we want... Closing.